Sming/Makefile-rboot.mk

```diff
@@ -18,7 +18,7 @@ RBOOT_SPIFFS_1   ?= 0x300000
 RBOOT_LD_0 ?= rom0.ld
 RBOOT_LD_1 ?= rom1.ld
 # esptool2 path
-ESPTOOL2 ?= esptool2
+ESPTOOL2 ?= c:\userdata\esp8266\esptool2\esptool2.exe
```

Comment:
You don't want to commit this change. If you want to change this variable for testing without editing the file just set it as an environment variable.